src/NHibernate/Dialect/PostgreSQLDialect.cs

```diff
@@ -329,6 +329,16 @@ public override string CurrentTimestampSelectString
 
 		public override string QuerySequencesString => "SELECT c.relname FROM pg_class c WHERE c.relkind = 'S'";
 
+		/// <summary>
+		/// Does this dialect supports distributed transaction? <c>false</c>.
+		/// </summary>
+		/// <remarks>
+		/// Npgsql since its version 3.2.5 version has race conditions: it fails handling the threading involved with
+		/// distributed transactions. This causes a bunch of distributed tests to be flaky with Npgsql. Individually,
+		/// they usually succeed, but run together, some of them fail. The trouble was not occuring with Npgsql 3.2.4.1.
+		/// </remarks>
+		public override bool SupportsDistributedTransactions => false;
```

Comment:
I don't quite agree with this change. It does support distributed transactions, but has some bugs. Also, not related, but, it seems that this property should be on a driver, not on a dialect.

Reply:
Sometimes that is more driver related, other times that just the db, whatever the driver, so then it is more dialect related.

Yes, this property is informational, it currently does not have any effect elsewhere than tests (or user code if some users do something with it).

From my viewpoint, Npgsql does no more properly support distributed transactions, because it can not even reliably close the connection right after disposing a distributed scope since Npgsql 3.2.5. Sometimes, a race condition will cause the connection to be still executing the second phase, and the close will blow up with an exception.

Reply:
Ok.